cfme/v2v/migrations.py

```diff
@@ -363,17 +371,51 @@ def plan_in_progress(self, plan_name):
             return False
 
 
+class MigrationDashboardView59z(MigrationDashboardView):
+    infra_mapping_list = InfraMappingList("infra-mappings-list-view")
+
+    @property
+    def is_displayed(self):
+        return (self.navigation.currently_selected == ['Compute', 'Migration'] and
+            (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0) and
+            (len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0))
+
+
+class InfrastructureMappingView(BaseLoggedInPage):
```

Comment:
This is an entire separate page, with many elements similar to what we had in `MigrationPlanRequestDetailsView` , so re-using some of those Paginator things by renaming those from `MigrationPlanRequestDetailsPaginator` to `v2vPaginator`, etc.